docs/software_requirements/timers.sdoc

```diff
@@ -26,12 +23,334 @@ UID: ZEP-SRS-4-2
 STATUS: Draft
 TYPE: Functional
 COMPONENT: Timers
-TITLE: Call functions in interrupt context
+TITLE: Timer definition at compile time
+STATEMENT: >>>
+The Zephyr RTOS shall provide a mechanism to define and initialize a timer at compile time.
+<<<
+RELATIONS:
+- TYPE: Parent
+  VALUE: ZEP-SYRS-18
+
+[REQUIREMENT]
+UID: ZEP-SRS-4-3
+STATUS: Draft
+TYPE: Non-Functional
```

Comment:
The status req is non functional because in first place this isn't good testable on it's own. And more detailed use of the status is specified in follow up regs.

Reply:
The way I understand your reply is, that basically non-functional here means it is some sort of aggregate RQT that has more fine-grained details in other RQTs. Probably, that's ok-ish enough for now.

Reply:
> The way I understand your reply is, that basically non-functional here means it is some sort of aggregate RQT that has more fine-grained details in other RQTs. Probably, that's ok-ish enough for now.

That is correct that was the intention here